### PR TITLE
AWS: add aws accounts for scalability tests

### PR DESCRIPTION
**Pull request by ameukam**
Related to:
  - https://github.com/kubernetes/test-infra/issues/29139

### CODE REVIEW

File: infra/aws/terraform/management-account/organization-accounts-boskos.tf [hunk 1]
```diff
@@ -204,3 +204,32 @@ module "k8s_infra_eks_e2e_boskos_010" {
     "service"     = "boskos"
   }
 }
+
+
+module "k8s_infra_e2e_boskos_scale_001" {
+  source = "../modules/org-account"
+
+  account_name               = "k8s-infra-e2e-boskos-scale-001"
```
Comment (xmudrii) [inline infra/aws/terraform/management-account/organization-accounts-boskos.tf:212]: I think we should call these accounts similar to boskos accounts, i.e. `k8s-infra-eks-e2e-boskos-scale-001`. This is considering these accounts are used by boskos in the EKS cluster.

Comment (ameukam) [reply infra/aws/terraform/management-account/organization-accounts-boskos.tf:212]: Not necessarily. The accounts could used on a different boskos config running on a non-EKS cluster (kops cluster, fargate, etc...).
We currently use static credentials for authn between Boskos and the accounts. That give the flexibility to reuse on a different environment.
I prefer to keep them generic.

**Comment by ameukam** [comment]
Accounts created:

```bash
aws organizations list-accounts --query "Accounts[?Status=='ACTIVE'].{Name: Name, Email: Email}" | grep 'scale'
k8s-infra-aws-boskos-accounts+scale-001@kubernetes.io	k8s-infra-e2e-boskos-scale-001
k8s-infra-aws-boskos-accounts+scale-002@kubernetes.io	k8s-infra-e2e-boskos-scale-002
```

**Comment by shyamjvs** [comment]
Thanks! Lgtm++

**Pull request merged**